src/Foundation/NSMetadataItem.cs

```diff
@@ -24,6 +24,16 @@ bool GetBool (NSString key)
 			return n == null ? 0 : n.DoubleValue;
 		}
 
+		nint GetNInt (NSString key)
+		{
+			var n = Runtime.GetNSObject<NSNumber> (GetHandle (key));
+#if XAMCORE_2_0
+			return n == null ? 0 : n.NIntValue;
```

Comment:
What about n?.NIntValue ?? 0;

If n is null, we get null therefore 0 will be used, else we do get the NIntValue value.